### PR TITLE
Update setup.py to coexist with stefanfoulis/django-phonenumber-field

### PR DESCRIPTION
**Pull request by francoisfreitag**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -19,10 +19,6 @@
         'phonenumberslite': ['phonenumberslite>=7.0.2'],
     },
     long_description=open('README.rst').read(),
-    author='Stefan Foulis',
-    author_email='stefan@foulis.ch',
```
Comment (jdufresne) [inline setup.py]: He is still the original author and so should remain here. Feel free to continue to remove maintainer though.

File: setup.py [hunk 2]
```diff
@@ -1,15 +1,15 @@
 # -*- coding: utf-8 -*-
-from phonenumber_field import __version__
 from setuptools import find_packages, setup
 
+from phonenumber_field import __version__
```
Comment (jdufresne) [inline setup.py]: While this line change is fine, it looks irrelevant to the change at hand.

**Comment by francoisfreitag** [comment]
Thanks for the review. Updated and ready for another review.

Changes:
- Add back author fields.
- Remove isort fix.
